Frameworks/CoreText/CTFont.mm

```diff
@@ -508,19 +601,19 @@ CGFloat CTFontGetUnderlineThickness(CTFontRef font) {
 }
 
 /**
- @Status Stub
+ @Status Interoperable
  @Notes
 */
 CGFloat CTFontGetSlantAngle(CTFontRef font) {
-    UNIMPLEMENTED();
-    return StubReturn();
+    return _DWriteFontGetSlantDegrees(font->_dwriteFontFace);
```

Comment:
hrmm idk, reference platform seems to just return default values rather than crashing, and while I think given the interface it's better to crash than silently accept bad input, I could see this posing a problem

---
In reply to: [80757662](https://github.com/Microsoft/WinObjC/pull/1039#discussion_r80757662) [](ancestors = 80757662,80733021)